### PR TITLE
fix scripts to collect from model zoo

### PR DESCRIPTION
**Pull request by emilydinan**
- Fix personachat scripts so that they collect from model zoo
- Add lines to language_model.py and persona_seq2seq.py to check for this.

### CODE REVIEW

**Comment by emilydinan** [comment]
1. Moved check for 'models:' prefix in model file into the `create_agent` function. Also added check for dict_file named similarly there. 
2. Small changes to language model sampling mode to make sure our sampling distribution norm does not blow up.
3. Added back a script for training Saizheng's profile memory model with parameters that produce better results.

File: projects/personachat/persona_seq2seq.py [hunk 1]
```diff
@@ -1664,6 +1668,7 @@ def _encode(self, xs, is_training=False):
             if type(self.decoder) != nn.LSTM:
                 hidden = hidden[0]
         else:
+            xes_packed = pack_padded_sequence(xes.transpose(0, 1), x_lens)
```
Comment (alexholdenmiller) [inline projects/personachat/persona_seq2seq.py]: if you use pack_padded_sequence then I think you'll need to use pad_packed_sequence on the encoder_output

Comment (alexholdenmiller) [reply projects/personachat/persona_seq2seq.py]: does this actually run as is?

**Comment by emilydinan** [comment]
@alexholdenmiller got rid of pack_padded for profile memory model

**Comment by emilydinan** [comment]
@jaseweston yes i checked before, still works!

**Pull request merged**